document/core/exec/instructions.rst

```diff
@@ -3682,65 +3682,71 @@ Memory Instructions
 
 9. Let :math:`\X{mem}_s` be the :ref:`memory instance <syntax-meminst>` :math:`S.\SMEMS[\X{sa}]`.
 
-10. Assert: due to :ref:`validation <valid-memory.copy>`, a value of :ref:`value type <syntax-valtype>` |I32| is on the top of the stack.
+10. Let :math:`\X{it}_d~\limits` be the :math:`\X{mem}_d.\MITYPE`.
 
-11. Pop the value :math:`\I32.\CONST~n` from the stack.
+11. Let :math:`\X{it}_s~\limits` be the :math:`\X{mem}_s.\MITYPE`.
```

Comment:
This could well have different limits than the other memory, so you need to subscript \limits, too.